build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -70,6 +70,8 @@
       <TestDirectory>$(RepoRoot)/test/</TestDirectory>
       <DotnetUnderTest>$(Stage2Directory)</DotnetUnderTest>
     </PropertyGroup>
+
+    <MakeDir Directories="$(TestPackagesDir)" Condition="!Exists('$(TestPackagesDir)')"/>
```

Comment:
(minor) is it possible to not make these test changes in this PR?  IMO it "clutters" what we are trying to accomplish here.